src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,102 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+@Entity
+@Table(name = "movie_sessions")
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne
+    @JoinColumn(name = "movie_id", referencedColumnName = "id")
+    private Movie movie;
+
+    @ManyToOne
+    @JoinColumn(name = "cinema_hall_id", referencedColumnName = "id")
+    private CinemaHall cinemaHall;
+
+    @Column(name = "start_time")
+    private LocalDateTime startTime;
+
+    @Column(name = "end_time")
+    private LocalDateTime endTime;
+
+    public MovieSession() {
```

Comment:
Don't create a default constructor when it is not needed. Since you are already providing a parameterized constructor and Hibernate does not require a default constructor if you are using field-based access, you can remove the default constructor.